sdk/textanalytics/ai-text-analytics/MIGRATION_v5_v6.md

```diff
@@ -0,0 +1,373 @@
+
+- Version 5 of the `@azure/ai-text-analytics` package _only_ supports Text Analytics service API version 3.0 and 3.1, and will not support the new Azure Cognitive Language Service.
+- Version 6 (beta) of the package supports Azure Cognitive Language Service API version "2022-04-01-preview", and future releases of this major version will support newer service API versions as well. It also drops support for the old Text Analytics API.
+
+This document provides instructions for updating your application code to the new major version 6 of the client library. In this document, the examples provided use TypeScript to provide type information, but all runtime behavior changes naturally apply to plain JavaScript as well.
+
+## Partial Migration (Side-by-Side)
+
+To avoid migrating an application all at once, major version 6 may be installed alongside major version 5 using a dependency alias. Either version may be aliased. For example, to install the new client library under an alias, add the following to the `dependencies` field of `package.json`:
+
+```javascript
+{
+    ...,
+    "dependencies": {
+        ...,
+        "@azure/ai-text-analytics": "^5.1.0",
+        "@azure/ai-text-analytics-new": "npm:@azure/ai-text-analytics@6.0.0-beta.1"
+    }
+}
+```
+
+With this configuration, imports from `"@azure/ai-text-analytics"` will import from major version 5, and imports from `"@azure/ai-text-analytics-new"` will import from major version 6. Of course, major version 5 could be aliased instead:
+
+```javascript
+{
+    ...,
+    "dependencies": {
+        ...,
+        "@azure/ai-text-analytics": "6.0.0-beta.1",
+        "@azure/ai-text-analytics-old": "npm:@azure/ai-text-analytics@^5.1.0"
+    }
+}
+```
+
+Then, the two packages may be used side-by-side, and an application may be migrated partially or over time:
+
+```javascript
+import { TextAnalyticsClient } from "@azure/ai-text-analytics-old";
+import { TextAnalysisClient } from "@azure/ai-text-analytics";
+```
+
+## Understanding the New Types
+
+In the new major version 6, several types have been introduced, renamed, restructured, and removed. This section describes the differences between the two versions.
+
+### Terminology
+
+Some terminology has changed to reflect the enhanced capabilities of the newest Azure Cognitive Language Service APIs. For starters, the service changed its name from Azure Text Analytics to Azure Cognitive Language Service and added advanced capabilities for answering questions and analyzing conversations. However, major version 6 will only support the text analysis capabilities that are provided by the Azure Cognitive Language Service. As a result, we've made the following broad changes to the terminology used throughout the client:
```

Comment:
terminologies?

Reply:
terminology itself is usually defined as a set of terms for a particular application so making it plural may not be needed here.